ts/system-test/test-start.ts

```diff
@@ -73,9 +73,9 @@ before(async () => {
       .reply(200, (request: RequestProfile, body: RequestProfile) => {
         tempUploadedProfiles.push(body);
       });
-  nock('https://accounts.google.com')
+  nock(/https:\/\/(accounts\.google\.com|www\.googleapis\.com)/)
```

Comment:
Are both URLs possible with 0.16.1?

Reply:
I think only googleapis.com is needed. Done.